.github/workflows/docker-release.yml

```diff
@@ -24,10 +24,37 @@ env:
   IS_PRERELEASE: ${{ github.event.release.prerelease || github.event.inputs.PRERELEASE }}
 
 jobs:
-# ===============================================================
-#   Building Dev Images
-# ===============================================================
+  # This workaround is needed, as it's not possible to reference env.FOOBAR directly at this stage
+  # - for ex. https://github.com/actions/runner/issues/1189
+  release-container-prereq:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.set-output-defaults.outputs.tag }}
+      tag_latest: ${{ steps.set-output-defaults.outputs.tag_latest }}
+    steps:
+    - id: set-output-defaults
+      run: |
+        echo "::set-output name=tag::${{ env.IS_PRERELEASE == 'true' && 'alpha' }}"
```

Comment:
what does it mean `${{ env.IS_PRERELEASE == 'true' && 'alpha' }}` ? what will it output when IS_PRERELEASE is true and when it's not?

Reply:
Added some comments in the code.
```
# If env.IS_PRERELEASE is true, set tag to alpha and do not enable tag_latest
# If env.IS_PRERELEASE is not true (aka false), don't set an extra tag and enable tag_latest
```